docs/interfaces.js

```diff
@@ -31,18 +31,27 @@
  * completes. The callback is given either:
  *
  * <ul>
- *   <li> a {@link Response} object; OR
- *   <li> an error object from the underlying <code>http</code> library.
+ *   <li> a successful {@link Response} object; or
+ *   <li> an error, one of:
+ *     <ul>
+ *       <li> the string <code>"timeout"</code>; or
+ *       <li> an error from the underlying <code>http</code> library; or
+ *       <li> a {@link Response} whose status is not <code>OK</code>.
+ *     </ul>
  * </ul>
  *
  * For example:
  *
  * <pre>
  *   googleMapsClient.geocode({...}, function(err, response) {
- *     if (err) {
- *       // Handle error.
- *     } else {
+ *     if (!err) {
  *       // Handle response.
+ *     } else if (err === 'timeout') {
+ *       // Handle timeout.
+ *     } else if (err.json) {
```

Comment:
If `err` is an instance of `Response`, do we expect `err.json` to always be true since it would be [a function that returns a promise](https://developer.mozilla.org/en-US/docs/Web/API/Response)? Or is it not the JavaScript `Response` (and perhaps a `Response` from another library) ?